Libraries/WebSocket/RCTWebSocketExecutor.m

```diff
@@ -69,9 +69,12 @@ - (void)setUp
   _injectedObjects = [NSMutableDictionary new];
   [_socket setDelegateDispatchQueue:_jsQueue];
 
-  NSURL *startDevToolsURL = [NSURL URLWithString:@"/launch-js-devtools" relativeToURL:_url];
-  [NSURLConnection connectionWithRequest:[NSURLRequest requestWithURL:startDevToolsURL] delegate:nil];
+  NSURL *startDevToolsURL = [NSURL URLWithString:@"/launch-chrome-devtools" relativeToURL:_url];
```

Comment:
Hmmm I think that got changed and then changed back by an upstream commit... will fix this.